connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -146,6 +146,7 @@
     private ChunkCache chunkCache;
     private EntityCache entityCache;
     private EntityEffectCache effectCache;
+    private PreferencesCache preferencesCache;
```

Comment:
Make this final and don't set it to null on close - we're going to do that with all the other fields when the most recent scoreboard PR is merged.

Reply:
addressed in 5dd4fa9656c116f99ea743dfdff89cfc1efe2209 and ba040ccd4a8fdaf37fbb11dcfa7b5294a6e3b1d4